lib/flatfile/config/QubitRelation-actor.yml

```diff
@@ -0,0 +1,9 @@
+# Columns that should be included in the actor relation iport
+columnNames:
+ - sourceAuthorizedFormOfName
+ - targetAuthorizedFormOfName
+ - category
```

Comment:
I think the relationship "type" needs to be added to this list.
![atom_250_authority_record_relationship_modal](https://user-images.githubusercontent.com/626135/79158534-5cdaf100-7d8b-11ea-95fa-601125331aa7.PNG)

Reply:
The "category" in this context is a UI nicety given it's a parent term of the type term... once you select the category it populates the select element for "type". So just specifying the type (labelled "category" in the existing CSV) works for importing.

Reply:
@mcantelon In that case I think calling the CSV column "relationship_type" (or similar) is better than calling it "category", to avoid confusing it with the UI "category" field (which isn't represented in the CSV template)

Reply:
I'm assuming here that "Relationship type" UI field actually sets the `realation.type_id` column in the database, and is the value we want in the CSV doc.

Reply:
Yeah, that does make sense. I've renamed it to "relationType".

Reply:
@mcantelon I'm not sure if Github is just showing me an old version of this file but I'm still seeing "- category" and not "- relationType" in this column list.